hera_mc/cm_dataview.py

```diff
@@ -26,7 +39,7 @@ def __init__(self, session=None):
         else:
             self.session = session
 
-    def ants_by_day(self, start, stop, time_step, output=None, station_types_to_check='default', output_date_format='jd'):
+    def connected_by_day(self, start, stop, time_step, output=None, station_types_to_check='default', output_date_format='jd', hookup_type='parts_hera'):
```

Comment:
docstring needs a Returns section. See here for the format: https://numpydoc.readthedocs.io/en/latest/format.html